### PR TITLE
Connected login to actions and api, getting user as props

### PR DESCRIPTION
**Pull request by johnandblue**

### CODE REVIEW

**Review by arol — CHANGES_REQUESTED** [review]
Please solve conflicts first

**Comment by arol** [comment]
Conflicts are still not resolved. 😞

**Comment by johnandblue** [comment]
@arol Conflicts resolved, ready to merge

**Comment by johnandblue** [comment]
Waiting for approval to merge

File: app/components/Swiper/SwiperEL.js [hunk 1]
```diff
@@ -5,57 +5,53 @@ import SwipeCards from 'react-native-swipe-cards';
...
+  handleNope = () => {
+    const movieId = this.props.movies[this.state.cardIndex].id;
     this.setState({ cardIndex: this.state.cardIndex + 1 });
-    this._swiper._goToNextCard();
+    this.props.dislikeMovie(movieId);
+    console.log(movieId);
   }
-
-
+  //
```
Comment (arol) [inline app/components/Swiper/SwiperEL.js]: Unused code

File: app/components/Swiper/SwiperEL.js [hunk 2]
```diff
@@ -118,7 +112,7 @@ class SwiperEL extends Component {
             </View>
           </TouchableOpacity>
           <TouchableOpacity
-            // onPress={like}
+            onPress={this.clicLike}
```
Comment (arol) [inline app/components/Swiper/SwiperEL.js:100]: Does this function exist?

**Review by arol — CHANGES_REQUESTED** [review]

**Comment by johnandblue** [comment]
@arol 
Unused code solved, 
SwiperEl is an outdated version, the new one is the branch that we are working actually

**Review by arol — APPROVED** [review]

**Comment by arol** [comment]
All yours, then.

**Pull request merged**